docs/run/rest-api-primitives.mdx

```diff
@@ -0,0 +1,490 @@
+---
+title: Primitives using REST API
+description: How to use the Sampler and Estimator primitives with Qiskit Runtime REST API.
+---
+
+# Primitives using REST API
+
+The steps in this topic describe how to run primitives workloads with REST API, explore the options you can use to configure them, and invoke them in any program of your choice.
+
+<Admonition type="note">
+    This documentation utilizes the `requests` module of python to demonstrate the Qiskit Runtime REST API. However, this workflow can be executed using any language/framework that supports working with REST APIs. The official documentation for the API Reference can be found [here](https://docs.quantum.ibm.com/api/runtime/tags/jobs).
+</Admonition>
+
+<span id="start-estimator"></span>
+## Estimator Primitive with REST API
+
+### 1. Initialize the account
+
+Because Qiskit Runtime `Estimator` is a managed service, you first need to initialize your account. You can then select the simulator or real system you want to use to calculate the expectation value. You can then select the IBM Quantum Compute resource of your choice to run your calculations on. 
+
+Details on how to initialize your account, view available backends and invalidate tokens can be found in this [topic](/start/setup-channel#access-ibm-quantum-systems-using-rest-api).
+
+### 2. Create a QASM circuit
+
+You need at least one circuit as the input to the `Estimator` primitive.
+
+
+* Define a QASM quantum circuit e.g. 
```

Comment:
```suggestion
Define a QASM quantum circuit e.g. 
```
This was rendering with a bullet point, which I assumed was not intentional? This change should fix that

Reply:
Thanks for catching that.